server/spec/content_delivery_network_spec.rb

```diff
@@ -70,4 +70,84 @@
     cdn.certificate['cert'].should == "test-cert"
   end
 
+  it 'should allow candlepin to create certificate serial' do
```

Comment:
The test is good to explain the current behavior. The reason I ask about pulling the value from the certificate is that if we used the value from inside the cert there would be no chance of it getting out of sync. I'll file it under ideas for future improvement since this is merely documenting the current API